docs/_data/product/tables.yml

```diff
@@ -0,0 +1,25 @@
+widths:
+  - class: .s-table--cell1
+    output: 8.3333333%
+  - class: .s-table--cell2
+    output: 16.6666667%
+  - class: .s-table--cell3
+    output: 25%
+  - class: .s-table--cell4
+    output: 33.3333333%
+  - class: .s-table--cell5
+    output: 41.6666667%
+  - class: .s-table--cell6
+    output: 50%
+  - class: .s-table--cell7
+    output: 58.3333333%
+  - class: .s-table--cell8
+    output: 66.6666667%
+  - class: .s-table--cell9
+    output: 75%
+  - class: .s-table--cell10
+    output: 83.3333333%
+  - class: .s-table--cell11
+    output: 91.6666667%
+  - class: .s-table--cell3
```

Comment:
This should be `s-table--cell100` or something, right?

Reply:
This follows the same pattern as `.grid--cell#`.